packages/react-core/src/components/Wizard/examples/WizardWithDrawer.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {
+  Button,
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerColorVariant,
+  DrawerHead,
+  DrawerPanelContent,
+  Wizard
+} from '@patternfly/react-core';
+
+export const WizardWithDrawer: React.FunctionComponent = () => {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isDrawerExpanded, setIsDrawerExpanded] = React.useState(false);
+  const [sectionGray, setSectionGray] = React.useState(false);
+  const [panelGray, setPanelGray] = React.useState(true);
+  const [contentGray, setContentGray] = React.useState(false);
+  /* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
We can remove any of these states if they aren't being used (looks like only isDrawerExpanded is)